pages/Documents/DeveloperTools/LivePersonFunctions/DevelopingwithFaaS/overview.md

```diff
@@ -51,12 +51,28 @@ The Payload Tab shows the example payload, which highlights the structure of the
 
 ### Environment Variables
 
-Using the relevant button from the Settings tab, you are able to add new Environment variables. Clicking the button will open a dialogue window where you can provide a name and a value for your new variable. As mentioned above, we have a set of reserved environment variables. The UI will inform you if you attempt to use them.
+Using the relevant button from the Settings tab, you are able to add new Environment variables. Clicking the button will open a dialogue window where you can provide a name and a value for your new variable.
 
 You can access the variable during runtime by using `process.env['YOUR_ENV']`. **Please be aware** that the value will be available in the form of a string. If the value is a number, you will have to parse it prior to using it. By using the trash icon, you can delete unwanted variables.
 
 **Developer discretion is advised**, please be sure not to use confidential data such as credentials or secrets in the environment variables as these are saved and available in text form to anyone with access to the account. Functions has a secret storage service that can be leveraged for this purpose.
 
+**Note**, they are certain reserved variables, that are used by the platform and therefore they are not available to the consumer. The UI will inform you if you attempt to use them. some of these variables are:
+`NODE_ENV`,
```

Comment:
Actually on second thought would actually remove the list, no need to share these details.

Reply:
updated